examples/prometheus.yml

```diff
@@ -0,0 +1,39 @@
+global:
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  evaluation_interval: 15s
+alerting:
+  alertmanagers:
+  - static_configs:
+    - targets: []
+    scheme: http
+    timeout: 10s
+    api_version: v1
+rule_files:
+- /etc/prometheus/alerts.yml
+scrape_configs:
+- job_name: prometheus
+  honor_timestamps: true
+  scrape_interval: 15s
+  scrape_timeout: 10s
+  metrics_path: /metrics
+  scheme: http
+  static_configs:
+  - targets:
+    - localhost:9090
+    - localhost:7979
+- job_name: json_exporter
+  honor_timestamps: true
```

Comment:
You probably don't want to honor timestamps, or expose timestamps, this leads to problems with staleness handling.

Reply:
Fixed!